datafusion/datasource/Cargo.toml

```diff
@@ -31,7 +31,7 @@ version.workspace = true
 all-features = true
 
 [features]
-parquet = ["dep:parquet", "tempfile"]
+parquet = ["tempfile"]
```

Comment:
I guess remove parquet feature from here as well?

Reply:
Proceeding by removing the `parquet` feature. Reasoning:
- `tempfile` is already included as part of the dependencies.
- There isn't any conditional code linked to the `parquet` features.

Thanks